Aspire Budgeting/Facilities/GoogleSheetsManager.swift

```diff
@@ -168,6 +168,7 @@ final class GoogleSheetsManager: ObservableObject {
   }
   
   private func createSheetsValueRangeFrom(amount: String,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/ca9f87ec120f1846bdadbae33cf063c210ffd1f2/Aspire Budgeting/Facilities/GoogleSheetsManager.swift#L170"></span>Function should have 5 parameters or less: it currently has 7<br />
<code>function_parameter_count</code> <code>GoogleSheetsManager.swift:170</code></td>
    </tr>
  </tbody>
</table>